sse/README.md

```diff
@@ -0,0 +1,63 @@
+# Server Sent Events Bindings
+
+This document defines how to describe Server Sent Events-specific information on AsyncAPI.
+
+See the [Server-Sent Events protocol specification][protocolSpecification].
+
+<a name="version"></a>
+
+## Version
+
+Current version is `0.1.0`.
+
+
+<a name="server"></a>
+
+## Server Binding Object
+
+This object MUST NOT contain any properties. Its name is reserved for future use.
+
+
+<a name="channel"></a>
+
+## Channel Binding Object
+
+When using Server Sent Events, the channel represents a single logical message stream flowing from server to client. 
+
+##### Fixed Fields
+
+Field Name | Type | Description
+---|:---:|---
+<a name="operationBindingObjectMethod"></a>`method` | string | The HTTP method to use when establishing the request. Its value MUST be `GET`.
```

Comment:
If the only allowed value is `GET`, can we just remove this field?

On the other hand, this whole object looks almost identical to the HTTP one. Shall we use the HTTP binding instead?

Reply:
The optional `last-event-id` request header is implicitly defined for SSE requests.

Actually, this header represents a cursor indicating progress into the SSE stream, allowing a disconnected client to recover from where they left off. So the logical SSE message stream is potentially spanning multiple HTTP requests.

If not abstracted out by `sse` protocol, how would the definition of acceptable HTTP headers that be handled by the proposed approach to use the HTTP binding for SSE message streams?

Reply:
> If the only allowed value is GET, can we just remove this field?
Yes. 👍